src/components/timecard/edit-shift-hours/EditShiftHours.spec.jsx

```diff
@@ -386,7 +416,7 @@ describe('EditShiftHours', () => {
     }
   );
 
-  test.each(['00:00', '08:00', '23:59', '04:26', '0000'])(
+  test.each(validTimes)(
```

Comment:
What exactly is the ruling on am/pm in the input box?
It isn't mentioned in the ticket, but the test suggests that its permissable. 
However it's not functioning correctly as per earlier comment. 
I think the reason this test managed to pass is because it's checking for the validation message but the service message is the one that's actually being displayed.

Reply:
As above, I've removed attempting to handle am/pm in the input box, we can add handling for them later